data/korath/korath outfits.txt

```diff
@@ -91,6 +92,32 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Liquid Sodium Cooler"
+	category "Systems"
+	cost 1660000
+	thumbnail "outfit/liquid sodium"
+	"mass" 34
+	"outfit space" -34
+	"energy consumption" 2
+	"cooling energy" 6
+	"active cooling" 151
+	description "This active cooling system pumps liquid sodium through the hottest parts of the ship. A secondary temperature control system draws a moderate amount of power in order to keep the sodium from freezing when the system is inactive."
```

Comment:
The awkward thing about this is that the ship itself isn't really all that hot - or the people would all be dead. It's the critical systems that are hot, and "overheating' a ship conceptually just means warming up its exterior hull enough that the heat-dumping into space is no longer efficient enough to maintain cooling to critical systems.

Also, since lithium doesn't melt until ~180 Celsius unless kept under pressure, and room temperature melting of Lithium requires pressurising to between 40 and 60 GPa (which is a lot for any significant volume of anything - like murder everything in the room and probably puncture the hull with flash-frozen shards if it ruptures) it might be good to call out something involved in that.

I would suggest adding base-heat generation to it equal to what would consume 2 points of cooling energy.
Now you could go and change it so that instead of using energy consumption, you add that energy to the cooling energy which would feel intuitive if heat made sense - but this would mean it costs no power when running below 0 heat. 
Though you could also split it, and have a mix of consumption + heat with suitable added cooling energy if you wanted to make a nod to that.
Either way add the same amount of heat generation to the active cooling amount, and then this is a net 0 impact on performance when you're above 0 heat, but if you're using this to augment passive cooling it then uses up some of the passive cooling buffer. 

Accordingly I would suggest maybe something more like this: 
```suggestion
	description "This active cooling system uses a liquid sodium cooling loop to extract heat from the hottest ship systems. A secondary system draws a moderate amount of power to continuously run the pump and maintain a minimum temperature in order to keep the sodium from freezing when the system is inactive."
	description "	As a side effect of this, the system actually produces heat when idling. This raises the base operating heat of the ship, but the efficiency benefits at high heat levels are well-realized."`
```

Reply:
It's sodium, not lithium, which looks like it has a melting point of 98 C.  Not that that makes a difference.

I'm concerned that, after all the many rounds of carefully balancing this outfit with the input of so many people, that adding more functionality for simulation reasons at this point would hazard upsetting that carefully-obtained disposition and result in further arduous rounds of balancing and debate.

Your lore suggestion is spot-on, though.

Reply:
Oh wow my brain really saw _**li**_ -quid sod- _**ium**_ and inserted lithium there... let me go clean my glasses.

Fair enough on the balance point - I'm like 98% sure the net impact could be made to be 0, but I fully respect that if the current balance works it may not be worth messing with just for lore fun, especially since _sodium's_ melting point is significantly lower.

If at some point in the future we end up with a suite of liquid-metal cooled active coolers across a few factions though, I do think it'd be a worthy exercise in that hypothetical future.

Happy the rest of it suited you. :)